src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -1140,7 +1140,11 @@ public override Task<ImplementationResult> ImplementationAsync(DelegatedPosition
             request,
             cancellationToken).ConfigureAwait(false);
 
-        if (response is null)
+        // Sometimes web tools will respond with a non-null response, of an array of diagnostics with a single element
+        // in it, but the actual diagnostics (and every other property) in that element are null. This confuses VS, and
```

Comment:
Can we be a bit more specific than "confuses"? Don't need the whole report but just if it throws an exception or something is fine.

Reply:
Error list is empty, and squiggles aren't shown. Still waiting to hear back from the editor team if its expected behaviour, or a bug on their end